redbot/cogs/downloader/repo_manager.py

```diff
@@ -2,17 +2,13 @@
 import functools
 import os
 import pkgutil
-import shutil
 from concurrent.futures import ThreadPoolExecutor
 from pathlib import Path
 from subprocess import run as sp_run, PIPE
 from sys import executable
 from typing import Tuple, MutableMapping, Union
 
-from discord.ext import commands
-
-from redbot.core import Config
```

Comment:
JSYK the `Config` import here was unused

Reply:
The commands import is used for type hinting, did you account for that?

Reply:
The commands import is replaced by `from redbot.core import commands`